collectd.spec

```diff
@@ -209,15 +209,15 @@
 # plugin tokyotyrant disabled, requires tcrdb.h
 %define with_tokyotyrant 0%{!?_without_tokyotyrant:0}
 # plugin write_kafka disabled, requires librdkafka
-%define with_write_kafka 0%{!?_without_write_kafka:0}
+%define with_write_kafka 0%{!?_without_write_kafka:1}
 # plugin write_mongodb disabled, requires libmongoc
 %define with_write_mongodb 0%{!?_without_write_mongodb:0}
 # plugin xmms disabled, requires xmms
 %define with_xmms 0%{!?_without_xmms:0}
 
 Summary:	statistics collection and monitoring daemon
 Name:		collectd
-Version:	5.5.0
```

Comment:
The source is having incorrect Version
$ grep -w Version collectd-5.5.1/contrib/redhat/collectd.spec
Version:	5.5.0

Its recently updated in src code
https://github.com/collectd/collectd/commit/9dbdf7b5ddeb11a7746009be443d63493408f613